src/front-end/src/App.js

```diff
@@ -13,7 +13,7 @@ function App() {
   return (
     <div className="App">
       <Button type="primary" onClick={callServer}>
-        ⏎········Call·the·server!⏎······!
```

Comment:
I have no idea why ESLint initially told me to format the text like that!! Thank you for fixing it, the way it was before looked so bad

Reply:
Haha yeah no worries, I don't know either =)